packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testRemoveAllChangeListItemsExcept.st

```diff
@@ -0,0 +1,15 @@
+testing-functionality
+testRemoveAllChangeListItemsExcept
+
+	| newMockChanges changeListItemsToKeep |
+	newMockChanges := CRBrowser getChangeListItemsFrom: 
+									(CRBrowser getChangeRecordsFrom: CRFakeFileStream mockChangesFileStream).
+	changeListItemsToKeep := OrderedCollection new.
+	changeListItemsToKeep
+		add: newMockChanges second;
+		add: newMockChanges fifth.
+
+	self changesBrowser removeAllChangeListItemsExcept: changeListItemsToKeep.
+	
+	self assert: (self changesBrowser changeListItems allSatisfy: [:each | 
+						changeListItemsToKeep includes: each])
```

Comment:
Shouldn't we also check if the other items got actually removed?
Right now, if `removeAllChangeListItemsExcept:` removed all items, this test would still pass.

Reply:
Fair point